### PR TITLE
[f] 987 - start twitterbot by a terminal command

### PR DESCRIPTION
**Pull request by anhhuy1605**
Note
--

* On root repo, install the twitter locally by using command `$ pip install -e .`
* Run `$ start_twitterbot`, it should output `Twitterbot started`

### CODE REVIEW

File: tests/integration_tests/test_start_twitterbot_command.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+except ImportError:
+    from mock import patch
+
+
+class StartTwitterBotCommandTestCase(TestCase):
+    def test_call_command(self):
+        with patch('twitterbot.start_twitterbot.TwitterBot.start') as mock_start:
+            CliRunner().invoke(start_twitterbot.main)
+
+            mock_start.assert_called_once_with()
```
Comment (tklarryonline) [inline tests/integration_tests/test_start_twitterbot_command.py]: This line can be placed outside of the `with` block :D

File: twitterbot/twitterbot.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, print_function
+
+from twitterbot.listeners.tweepy_listener import TweepyListener
+from twitterbot.utils.tweepy_utils import TweepyUtils
+
+
+class TwitterBot:
```
Comment (tklarryonline) [inline twitterbot/twitterbot.py]: To make it Py2 compatible, extends from `object` instead...

File: twitterbot/utils/tweepy_utils.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+import tweepy
```
Comment (tklarryonline) [inline twitterbot/utils/tweepy_utils.py]: Have you tried the following yet, before applying `__future__` ?

```py
from tweepy import OAuthHandler, API
```

**Comment by tklarryonline** [comment]
The module `twitterbot.twitterbot`'s coverage had decreased a lot

**Comment by anhhuy1605** [comment]
We added `twitterbot.twitterbot` and `utils.tweepy_utils` to test if running from command line raise any problem, so the code there is not ready yet and we didn't write any tests for them.

File: twitterbot/utils/tweepy_utils.py (hunk 3 above)
Comment (anhhuy1605) [inline twitterbot/utils/tweepy_utils.py]: The `2to3` translated that line to `from .tweepy import OAuthHandler, API` 😞 . I think we should change to Python 2 environment, that should be easier to write compatible code

File: twitterbot/twitterbot.py (hunk 2 above)
Comment (anhhuy1605) [inline twitterbot/twitterbot.py]: Yes, will change all code to Python 2 syntax

File: tests/integration_tests/test_start_twitterbot_command.py (hunk 1 above)
Comment (anhhuy1605) [inline tests/integration_tests/test_start_twitterbot_command.py]: Yes, but I like to keep it inside `with` block 😄

**Pull request merged**